### PR TITLE
[TR] PIM-4153 : Use of MySQL session storage

### PR DESCRIPTION
**Pull request by jmleroux**
Use of a non blocking session storage allows real async ajax calls.

| Q                    | A
| -------------------- | ---
| Bug fix?             |n
| New feature?         |y
| BC breaks?           |n
| CI currently passes? |y
| Tests pass?          |y
| Scenarios pass?      |y
| Checkstyle issues?*  |n
| PMD issues?**        |n
| Changelog updated?   |y
| Fixed tickets        |PIM-4153
| DB schema updated?   |y
| Migration script?    |y
| Doc PR               |n

### CODE REVIEW

File: src/Pim/Bundle/InstallerBundle/Command/DatabaseCommand.php [hunk 1]
```diff
@@ -110,6 +112,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+    {
+        $output->writeln('<info>Create session table</info>');
+
+        $sessionTableSql = "CREATE TABLE pim_session (
+                `sess_id` VARBINARY(128) NOT NULL PRIMARY KEY,
+                `sess_data` BLOB NOT NULL,
+                `sess_time` INTEGER UNSIGNED NOT NULL,
+                `sess_lifetime` MEDIUMINT NOT NULL
+            ) COLLATE utf8_bin, ENGINE = InnoDB";
+
```
Comment (jmleroux) [inline src/Pim/Bundle/InstallerBundle/Command/DatabaseCommand.php]: Structure of a SF 2.6 session table. Compatible with 2.3 version, only the last column will always be 0.

Comment (nidup) [inline src/Pim/Bundle/InstallerBundle/Command/DatabaseCommand.php]: we should add a migration script in https://github.com/akeneo/pim-community-standard/tree/master/upgrades/schema

**Comment by nidup** [comment]
sounds good for me, would like a GTM of @BitOne on this one ;)

File: src/Pim/Bundle/InstallerBundle/Command/DatabaseCommand.php (hunk 1 above)
Comment (jjanvier) [inline src/Pim/Bundle/InstallerBundle/Command/DatabaseCommand.php]: you can put it in an /upgrades folder in this repo at the moment

File: app/config/pim_parameters.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ parameters:
     websocket_host:       "127.0.0.1"
     websocket_port:       8080
 
-    session_handler:      ~
+    session_handler:      session.handler.pdo
```
Comment (BitOne) [inline app/config/pim_parameters.yml]: Not sure that it should be here or in config.yml.
Not exactly a parameter that needs to be changed by the user. And in fact, maybe not suitable for use changes, as it has effect on PIM performances if switched back to file storage.

File: src/Pim/Bundle/InstallerBundle/Command/DatabaseCommand.php (hunk 1 above)
Comment (jmleroux) [inline src/Pim/Bundle/InstallerBundle/Command/DatabaseCommand.php]: Migration aded

**Comment by sumbobyboys** [comment]
GTM

**Comment by jmleroux** [comment]
ping @BitOne

**Pull request merged**